drivers/filesystems/fastfat/finfo.c

```diff
@@ -831,6 +835,10 @@ VfatSetRenameInformation(
                              (vfatFCBIsDirectory(FCB) ?
                               FILE_NOTIFY_CHANGE_DIR_NAME : FILE_NOTIFY_CHANGE_FILE_NAME),
                              FILE_ACTION_RENAMED_OLD_NAME);
+
+            if (IsDots(NewFile))
```

Comment:
A single check right after the DPRINT in L805/809 above should do the trick I think.
`vfatReportChange` is a side-effect that we don't want to see unless the file is actually going to get moved.

Reply:
Done. Thanks.